src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+
+Lift and Shift is a commonly used approach. However, it may not be the right approach if a strong application service continuity during the migration is required. It may be a viable method if application downtime is permitted.
+
+{{site.data.alerts.callout_info}}
+The information on this page assumes you have already reviewed the [Migration Overview](migration-overview.html).
+{{site.data.alerts.end}}
+
+## Pros and Cons
+
+On the spectrum of different data migration strategies, 'lift and shift' has the following pros and cons. The terms "lower" and "higher" are not absolute, but relative to other approaches.
+
+Pros:
+
+- Conceptually straightforward
+- Less complex: if you can afford some downtime, the overall effort will usually be lower, and the chance of errors is lower
+- Lower time start-to-finish: in general, the more downtime you can afford, the shorter the overall migration project timeframe can be
+- Lower technical risk: it does not involve running multiple systems alongside each other for an extended period of time
+- Easy to practice dry runs of import/export using testing/non-production systems
+- Good import/export tooling is available (e.g. external tools like: [AWS Database Migration Service (DMS)](aws-dms.html), [Qlik Replicate](qlik.html), [Striim](striim.html); or internal tools like [`IMPORT INTO`](import-into.html), [`COPY`](copy-from.html), [`cockroach userdata`](cockroach-userfile-upload.html))
+- If your application already has regularly scheduled maintenance windows, your customers will not encounter application downtime
+
+Cons:
+
+- Higher project risk: It either works or does not work; once you start, you have to finish or rollback
+- Application service continuity requirements must be relaxed (that is, application downtime or degradation may be needed)
+- Less control of the pace (the project *has* to be completed to meet a given downtime / service degradation window)
+
+## Process design considerations
+
+{{site.data.alerts.callout_info}}
+The high-level considerations in this section only refer to the data loading portion of your migration. They assume you are following the steps in the overall migration process described in [Migrate your Database to CockroachDB](migration-overview.html).
+{{site.data.alerts.end}}
+
+Keep in mind the following considerations when designing a 'Lift and Shift' data migration process.
+
+- [Decide on your data migration tooling](#managed-migration)
+- [Decide which data formats you will use](#data-formats)
+- [Design a restartable process](#restartable)
+- [Design a scalable process](#scalable)
+
+<a name="managed-migration"></a>
+
+### Decide on your data migration tooling
+
+If you plan to do your bulk data migration using a managed migration service, you must have a secure, publicly available CockroachDB cluster. CockroachDB supports the following migration services:
```

Comment:
```suggestion
If you plan to do your bulk data migration using a managed migration service, you must have a secure, publicly available CockroachDB cluster. CockroachDB supports the following [third-party migration services](third-party-database-tools.html#data-migration-tools):
```

Reply:
oh nice, I did not know about that page - thanks!